docs/abi-spec.rst

```diff
@@ -97,7 +97,7 @@ We distinguish static and dynamic types. Static types are encoded in-place and d
 * ``bytes``
 * ``string``
 * ``T[]`` for any ``T``
-* ``T[k]`` for any dynamic ``T`` and any ``k > 0``
+* ``T[k]`` for any dynamic ``T`` and any ``k >= 0``
```

Comment:
Why is `T[0]` valid in the first place?

Reply:
Why not? It reduces the amount of exceptional cases.